app/mailers/candidate_mailer.rb

```diff
@@ -85,6 +85,15 @@ def new_offer_decisions_pending(application_choice)
     new_offer(application_choice, :decisions_pending)
   end
 
+  def reference_received(reference)
```

Comment:
Is there any test around this?

Reply:
Yeah, in the integration test: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1392/files#diff-228a1e3080215c6cca421e46eeb296aa

Do you think it's enough?

Reply:
Oh yeah, its a pretty simple mail!